server/app.ts

```diff
@@ -32,43 +71,48 @@ new MathigonStudioApp()
     })
     next()
   })
-  .get('/course/:course/:section', async (req, res, next) => {
-    const course = getCourse(req.params.course, req.locale.id)
-    const section = course?.sections.find(s => s.id === req.params.section)
+  .get('/account', async (req, res) => {
+    const userMockData = {
+      name: 'Russell Huffman',
```

Comment:
Can we try with a super lengthy name to see how the design/code handle longer user names?

Reply:
In the new commit, I added longer mock text and fixed how it is shown. Also, I limited the text that could be shown so it doesn't break the layout at all.